### DISCUSSION TITLE
Soroban Quest Chapter Submissions

### DISCUSSION THREAD

**Discussion by RDCrypto** [stellar/sorobanathon#151] [General]
My 7 Chapter Soroban Quest submissions

**Comment by RDCrypto** [comment]
Love what Stellar is doing to get everyone involved in questing and creating!  The story is really fun. Hope there are more future chapters for the quest!  I am not a real artist but have done all the Stellar quests going way back to the beginning and wanted to participate in this part also! Kudos to the Stellar team for putting this all together! By far the best community going!

**Comment by RDCrypto** [comment]
#Prologue (0)
https://litemint.com/items/GAX2IXBZ6HNTTB7LUORCQM2UZ3LZ3MO4DT4RA3LCY6PW4CBWPK52K7DN/Camp01
#The Temple (1)
https://litemint.com/items/GD4MX2RPZZRTKFIWQMWKQVTYK23JDZD4U34UNFZC2HABMVSC75KTEDGV/Temple1
#Lava Snakes (2)
https://litemint.com/items/GB5SNCBTN5XQKYSLB6XUFJJQPKD5HELXJ4ZLKEF3L4UVDNCUXRQSXYID/LavaSnakes1
#The Wooden Door (3)
https://litemint.com/items/GDJVZSB4PEZR7YFVQWVQFV3DISS5HNKVZFA3JPONQVY2FT6V6GLT77GL/WoodenDoor1
#The Circle Room (4)
https://litemint.com/items/GCV4MYJRVVKG5I4DAXK7Q3YXBNFJQYNHC3GQ57GSVB4HAQSAWBNVQL42/CircleRoom1
#The Dark Cavern (5)
https://litemint.com/items/GCNERWJGRZL3K6JNKHJ7M45Z3ZK2EZYW2EO3E5X2BS6VDIR2M3UUZC4U/DarkCavern1
#The Golden Soroban (6)
https://litemint.com/items/GD5SXC2PIQOGSC3U6A7YWTUJLPY3DIMBTDYNVCYVHYYUINNFEXBU2RDP/GoldenSoro1

**Comment by stellarjohn** [comment]
Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
